sansio_lsp_client/events.py

```diff
@@ -79,3 +92,80 @@ class WillSaveWaitUntilEdits(Event):
 class PublishDiagnostics(ServerNotification):
     uri: str
     diagnostics: t.List[Diagnostic]
+
+
+# NEW ##################
+""" Hover:
+    * contents: MarkedString | MarkedString[] | MarkupContent;
+    * range?: Range;
+"""
```

Comment:
This should be a comment, not a string. Currently it's not a docstring, because a docstring has to be the first statement in a function, class or file.

Reply:
There are many other strings like this too